File: devops-scm-api/src/main/java/com/tencent/devops/scm/api/pojo/Signature.java
package com.tencent.devops.scm.api.pojo;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * 代表代码提交者
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Signature {
    private String name;
    private String email;
    private String avatar;
}

File: devops-scm-api/src/main/kotlin/com/tencent/devops/scm/api/pojo/Signature.kt
package com.tencent.devops.scm.api.pojo

/**
 * 代表代码提交者
 */
data class Signature(
    // 提交者名称
    val name: String,
    // 提交者邮箱
    val email: String? = null,
    // 提交者头像
    val avatar: String? = null
)
